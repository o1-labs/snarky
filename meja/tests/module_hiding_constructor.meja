module A = {
  type t = X;
};

module B = {
  module A = {
    type u = Y;
  };
};

open B;

let x = A.Y;

/* Expect an error, A should be out of scope. */
let y = A.X;
