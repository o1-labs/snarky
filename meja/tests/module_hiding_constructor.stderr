File "tests/module_hiding_constructor.meja", line 16, characters 8-11:
Error: Unbound constructor A.X.
